code/client/src/sdk/entities/c_vehicle.cpp

```diff
@@ -31,8 +31,8 @@ namespace SDK {
         hook::this_call(gPatterns.C_Vehicle__SetHandbrake, this, force, arg2);
     }
 
-    void C_Vehicle::SetEngineOn(bool arg1, bool arg2) {
-        hook::this_call(gPatterns.C_Vehicle__SetEngineOn, this, arg1, arg2);
+    void C_Vehicle::SetEngineOn(bool on, bool arg2) {
+        hook::this_call(gPatterns.C_Vehicle__SetEngineOn, this, on, arg2);
```

Comment:
We don't exactly know what are the two arguments for, sometimes the game pass only the first one and sometimes both. Lets not make assumptions.